core/src/dsnp/api_types.rs

```diff
@@ -50,17 +67,35 @@ pub trait Config {
 }
 
 /// Encapsulates all the keys and page data that needs to be retrieved from chain
-pub struct Import<E: EncryptionBehavior> {
+pub struct ImportBundle<E: EncryptionBehavior> {
```

Comment:
I think this might be useful for importing keys and pages to verify a PRI friendship but I don't think we can populate the pages and keys at the same time for private pages since we will need to decrypt connections to be able to know which public keys we will need to fetch for PRI generation

Reply:
The keys in the import bundle are the owner's keys needed for decryption, not other users' public keys. I'll update the doc comment to be more clear on that.